interfaces/automated-testing/scd/design/operational-intent-format/uas_flight_auth_uas_serial_number.plantuml

```diff
@@ -0,0 +1,15 @@
+@startuml
+center header Operational Intent Format
+title UAS Flight Auth uas_serial_number
+
+participant "Test Driver" as TD
+participant "USS under test" as USS
+
+== Invalid ==
+TD --> USS: PUT /inject/operational_intent\nwith invalid uas_serial_number
+TD <-- USS: 400 Bad Format
+
+== Valid ==
```

Comment:
Just a note: I think this is a strong candidate for combining when more tests are defined (no reason to repeat injection of a fully-valid operational intent once for valid time range, again for valid geography, again for valid registration number, again for valid state, etc)

Reply:
Sounds good to me. I will add the next tests in the same file. Should we envision splitting it in two sets of tests? On one end, ASTM requirements (time range, valid geography, etc.) and on the other hand, U-Space requirements related to the flight auth?